addons/web/static/src/js/libs/bootstrap.js

```diff
@@ -96,6 +96,16 @@ $.fn.scrollspy.Constructor.prototype.refresh = function () {
     }
 };
 
+/* In some cases, we need to keep the first element of navbar selected (e.g 'Table of Content') */
+
+const bootstrapSpyProcessFunction = $.fn.scrollspy.Constructor.prototype._process;
+$.fn.scrollspy.Constructor.prototype._process = function () {
+    bootstrapSpyProcessFunction.apply(this, arguments);
+    if (this._activeTarget === null && this._config.active) {
```

Comment:
Maybe use a config name better than "active"? Like `alwaysKeepOneActive`? Would be better to understand and grep.